src/lib/utils/renderWithLoadProgress.tsx

```diff
@@ -3,11 +3,18 @@ import React from "react"
 import LoadFailureView from "lib/Components/LoadFailureView"
 import Spinner from "lib/Components/Spinner"
 import { RenderCallback } from "lib/relay/QueryRenderers"
+import { NetworkError } from "lib/utils/errors"
 
 export default (Component: React.ReactType, initialProps: object = {}): RenderCallback => {
   let retrying = false
   return ({ error, props, retry }) => {
     if (error) {
+      const networkError: NetworkError = error as any
+      if (networkError.response) {
+        const data = (networkError.response as any)._bodyInit
+        console.error(`Metaphysics Error: ${error.message}\n`, JSON.parse(data))
```

Comment:
Do we really need to parse the data instead of just including it as-is? My paranoia is telling me that I wouldn’t want us to get into a situation where there are exceptions during error reporting.

Reply:
I guess it depends, it makes it much easier to debug at dev time (because it's an object you an introspect ) otherwise it's a bunch of escaped JSON  - I can add some null checking around it, which I think is reasonable

Reply:
Fair enough 👍